### ISSUE TITLE
Not running in newer versions of R

### ISSUE THREAD

**Issue by MurrayLogan** [philliplab/EpitopeMatcher#73]
Hi Phil, 

For some reason I'm not able to run EpitopeMatcher anymore from an R session. 

Please could you help with this. 

Thanks!
Murray.

**Comment by MurrayLogan** [comment]
I get the following errors when running the code:

> library(devtools)
Error in library(devtools) : there is no package called ‘devtools’
> install_github('rstudio/shiny')
Error: could not find function "install_github"
> install_github('philliplab/EpitopeMatcher')
Error: could not find function "install_github"
> ```
Error: attempt to use zero-length variable name

**Comment by philliplab** [comment]
Hey

The issue you are reporting here is that devtools is not on your system.

Run this command in R:
 
install.packages('devtools')

and try again.

There might be other issues, I will install it today to see if there are some fixes that are required.

**Comment by MurrayLogan** [comment]
Thanks Phil. 

I eventually got it to run by installing Biostrings from: https://bioconductor.org/packages/release/bioc/html/Biostrings.html using these commands: 
if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")

BiocManager::install("Biostrings")

It seems there has been an update to the location of source code for this. 

I then ran the install.packages('devtools') command from R version 3.6.3 and it seemed to do the trick - the other commands ran fine once I overcame these two issues. 

Appreciate the prompt response,
Murray.
